src/common/components/eventForm.js

```diff
@@ -19,6 +20,7 @@ const radioStyle = {
 class EventForm extends Component {
   componentDidMount = () => {
     this.props.form.validateFields()
+    document.title = this.pageHeader()
```

Comment:
I think you can do ` document.title = this.props.edit ? "Edit Event" : "Create New Event"